Adyen/ApiSerialization/SaleToPoiMessageSerializer.cs

```diff
@@ -64,6 +69,34 @@ public SaleToPOIResponse Deserialize(string saleToPoiMessageDto)
             return deserializedOutputMessage;
         }
 
+        public SaleToPOIRequest DeserializeTerminalNotification(string terminalNotificationJson)
```

Comment:
```suggestion
        public SaleToPOIRequest DeserializeNotification(string terminalNotificationJson)
```
Maybe something like this because the class is already for terminal api only